service/header/sync.go

```diff
@@ -254,40 +301,88 @@ func (s *Syncer) incoming(ctx context.Context, maybeHead *ExtendedHeader) pubsub
 var requestSize uint64 = 512
 
 // syncTo requests headers from locally stored head up to the new head.
-func (s *Syncer) syncTo(ctx context.Context, newHead *ExtendedHeader) error {
+func (s *Syncer) syncTo(ctx context.Context, newHead *ExtendedHeader) {
 	head, err := s.store.Head(ctx)
 	if err != nil {
-		return err
+		log.Errorw("getting head during sync", "err", err)
```

Comment:
Is there a reason you don't return an error anymore?

Reply:
Yes. The goal was to have all the related logouts in one place. The error was returned previously only to be logged out, but now I log all the errors inside `syncTo`, so there is no point to return them anymore.